### ISSUE TITLE
Add responsive calendar to WP Theme

### ISSUE THREAD

**Issue by djmosfett** [dsacincy/wp-dsa-cincy#3]
This issue does what it says on the tin. As discussed, there are two major directions we can go:

1.  Embed the Google Calendar directly. A method to handle this is here: 
https://premium.wpmudev.org/blog/responsive-calendar-wordpress/

2.  Add a WP plugin. Major candidates are:

- Simple Calendar: https://wordpress.org/plugins/google-calendar-events/

- Tockify: https://tockify.com/

I suppose another issue should be "pick a method."

**Comment by djmosfett** [comment]
A new method for technique 2: https://theeventscalendar.com/product/wordpress-events-calendar/
This seems to be the way that Seattle DSA does it.

**Comment by djmosfett** [comment]
Updates: 

Method 1: 
Child theme creation is potentially involved. I will be working through this in parallel to the easier option of Method 2. 

Method 2: 
1. Events Calendar requires $$$ to import Google Calendar. So, we're axing that option. 
2. Simple Calendar does not require $$$ to import, _but_ it does require an API key. For the purposes of our site, this should be done by the holder of the dsacincy@gmail account, since that's our central email. 
**TODO**: Ask Charlie to follow the API process & provide us with the API key for the calendar.
